portfolio/src/main/webapp/style.css

```diff
@@ -1,108 +1,269 @@
-#content {
-    margin-left: auto;
-    margin-right: auto;
-    width: 650px;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    color: #e9f5ff;
-    flex-direction: column;
-    font-family: Verdena, Geneva, sans-serif; 
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: normal; 
+  color: #fdf7f7;
+  line-height: 1.3;
 }
 
-#greeting-container {
-    margin-top: 20px;
+.content {
+  max-width: 800px;
+  padding: 0 24px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
 
-.picture{
-    width: 20em;
-    height: 10%;
+.about,
+.skills,
+.contact,
+.comment {
+  padding-top: 1rem;
 }
 
-.column {
-    float: left;
-    width: 33.33%;
-    padding: 51px;
+.header,
+.footer {
+  background: #2c3e50;
+  z-index: 1;
+  top: 0;
+  position: sticky;
 }
 
-.row::after {
-    content: "";
-    clear: both;
-    display: table;
+.skills,
+.about {
+  background: #1abc9c;
 }
 
-* {
-    -webkit-box-sizing: border-box;
-    -moz-box-sizing: border-box;
-    box-sizing: border-box;
+.header ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
 }
 
-#header { 
-    display: flex;
-    align-items: center;
-    justify-content: center;
+.header ul li a, .footer div a, .header h1 a {
+  display: inline-block;
+  margin: 0 8px;
+  text-decoration: none;
+  color: #fdf7f7;
+}
+
+.picture {
+  max-width: 25%;
```

Comment:
Avoid % on width properties with the exception of 100%. Specify the maximum size in pixels.